content/automate/ManageIQ/Infrastructure/VM/Retirement/Email.class/__methods__/vm_retire_extend.rb

```diff
@@ -15,6 +15,7 @@ def initialize(handle = $evm)
               end
 
               def main
+                @handle.log("warn", "[DEPRECATION] This method will be deprecated. Please use similarly named method from System/Notification/Email class.”)
```

Comment:
@billfitzgerald0120 The quote character on the end of this line is not a standard quotation mark, it is stylized which is causing the rubocop warnings.